server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -394,6 +404,12 @@ private static long jitter() {
         * TabletServer.TIME_BETWEEN_LOCATOR_CACHE_CLEARS);
   }
 
+  /**
+   * If the user has set {@link Property#TSERV_WRITE_THREADS_MAX} then return the semaphore that
+   * controls the number of write threads. If the user has not set the value then return
+   * Optional.empty(), which is essentially a no-op.
+   */
```

Comment:
Is this comment in the right place?

Reply:
Nope. Another leftover scrap. I will drop it.